RELEASING.md

```diff
@@ -95,64 +101,23 @@ Go to the PR created and:
 
 1. Create a git tag for the new commit on the release branch and push it:
 
-```
-git tag v1.30.3
-git push --tags
-```
+    ```sh
+    git tag v1.30.3
+    git push --tags
```

Comment:
I'd suggest pushing only the newly created tag. If a user has other tags locally we probably don't want them all pushed.

Reply:
Yep, 100% this.